library/core/src/cell.rs

```diff
@@ -2092,6 +2112,17 @@ impl<T> const From<T> for SyncUnsafeCell<T> {
 //#[unstable(feature = "sync_unsafe_cell", issue = "95439")]
 impl<T: CoerceUnsized<U>, U> CoerceUnsized<SyncUnsafeCell<U>> for SyncUnsafeCell<T> {}
 
+// Allow types that wrap `SyncUnsafeCell` to also implement `DispatchFromDyn`
+// and become object safe method receivers.
+// Note that currently `SyncUnsafeCell` itself cannot be a method receiver
+// because it does not implement Deref.
+// In other words:
+// `self: SyncUnsafeCell<&Self>` won't work
+// `self: SyncUnsafeCellWrapper<Self>` becomes possible
+#[unstable(feature = "dispatch_from_dyn", issue = "none")]
+//#[unstable(feature = "sync_unsafe_cell", issue = "95439")]
```

Comment:
Looks like a stray line here?

Reply:
I copied this from the above `CoerceUnsized` impl.
I assumed this was meant to signal that there are two unstable features involved here.

Reply:
Yeah there has been a longstanding feature request for unstable APIs to be able to be covered by multiple tracking issues / feature gates. For example if there's a `NonZeroI256` type, and an unstable `wrapping_neg` inherent method on it (which is also unstable on a bunch of stable integer types), the method should be covered by gated on _both_ features. We have had a problem in the standard library in the past of accidentally over-stabilizing in similar situations. If `NonZeroI256` were stabilized and its `wrapping_neg` was gated on the `NonZeroI256` feature instead of the `wrapping_neg` feature, it might inadvertently get stabilized during the `NonZeroI256` stabilization without adequate review.

I thought there was a feature request issue filed for this but I wasn't able to find it just now.

The way this is written in the PR is the best we can currently do.